### DISCUSSION TITLE
Snow indicator when Copilot is generating a response

### DISCUSSION THREAD

**Discussion by villesau** [community/community#47969] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Would be good if there would be some sort of indicator showing when Copilot is thinking for a response. Not it is not possible to know if Copilot has given up or not. This is at least the case on IntelliJ.

Below an example from Chat-GPT showing it is generating a response. See the two dots (goes from one to three dots while it is work in progress):

<img width="380" alt="Screenshot 2023-02-19 at 17 17 51" src="https://user-images.githubusercontent.com/2647016/219957214-5dd5c9ab-c156-4b32-ad41-80f637081fe9.png">

**Comment by iamcjwilliams** [comment]
@villesau Thank you for your feedback, our copilot team has been notified
